relay-quotas/src/cache.rs

```diff
@@ -480,12 +480,9 @@ mod tests {
         assert_eq!(cache.check_quota(q1, 5), Action::Accept);
         assert_eq!(cache.check_quota(q1, 1), Action::Check(6));
 
-        // 31 remaining -> 3 (10%), consumption still under limit threshold (70),
-        // but maximum cached consumption would be *above* the threshold, this is currently
-        // explicitly not considered (but this behaviour may be changed in the future).
+        // 31 remaining -> 3 (10%), consumption still under limit threshold (70)
```

Comment:
This comment doesn't really relate to the assertion below. Maybe something like
```suggestion
        // 31 remaining -> 3 (10%), consumption still under limit threshold (70), but consumption + local use
        // exceeds the threshold.
```